src/test/groovy/com/google/appengine/task/internal/KickStartParamsBuilderTest.groovy

```diff
@@ -77,6 +77,31 @@ class KickStartParamsBuilderTest extends Specification {
             params.get(6) == "$KickStartParamsBuilder.JVM_FLAG=-Dappengine.user.timezone=UTC"
             params.get(7) == '/dev/main/war'
     }
+    
+    def "Build parameters for minimal arguments plus disable update check, httAddress, additional JVM flags and generatedDir"() {
```

Comment:
I think this test can just be merged with the previous one. The value is in really testing the generatedDir